src/pages/signin/LoginForm.js

```diff
@@ -94,6 +96,13 @@ function LoginForm(props) {
         [props.account, props.closeAccount, input, setFormError, setLogin],
     );
 
+    /**
+     * Enables experimental "memory only keys" mode in Onyx
+     */
+    const setEnableMemoryOnlyKeys = () => {
+        window.enableMemoryOnlyKeys();
+    };
```

Comment:
NAB: I would prefer this defined outside of the component since it doesn't depend on anything from it.